### PR TITLE
[spaceship] Receive SMS code for 2FA without the need to bypass trusted device

### PR DESCRIPTION
**Pull request by mariannegru**
### Checklist
- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.

### Motivation and Context

With this changes we do not need for the step # Bypass trusted device and use SMS for verification from [2 Step Verification](https://github.com/fastlane/fastlane/tree/master/spaceship#2-step-verification)

### Description

I added a method to show a selection of trusted phones to select from for sending the code required on the next step in 2fa login

Using an account with 2fa enabled I executed `fastlane spaceauth -u xxxx` and I was prompted with a list of phones to select from. After selecting the phone number I received the expected cookie

### CODE REVIEW

**Comment by mariannegru** [comment]
> I signed it!

I signed it!

File: spaceship/lib/spaceship/two_step_client.rb [hunk 1]
```diff
@@ -120,6 +122,37 @@ def self.spaceship_session_env
       ENV["FASTLANE_SESSION"] || ENV["SPACESHIP_SESSION"]
     end
 
+    def select_phone(response)
+      puts("Please select a phone to send code too:")
```
Comment (RishabhTayal) [inline spaceship/lib/spaceship/two_step_client.rb]:
```suggestion
      puts("Please select a phone to send code to:")
```

**Review by RishabhTayal — CHANGES_REQUESTED** [review]

**Comment by mariannegru** [comment]
@RishabhTayal can you please accept cla?

**Comment by joshdholtz** [comment]
@mariannegru Thanks for making this PR! I'm actually going to close this PR in favor of #13507 (which was made first). I appreciate the time put in to make this PR though ❤️

**Comment by janpio** [comment]
PR #14011 has been merged and this will be included in the next release 👍 Thanks for you work on this @mariannegru!

**Pull request closed without merging**